src/renderer/services/dex.js

```diff
@@ -29,7 +29,7 @@ const SIGNATUREREQUESTTYPE_WITHDRAWSTEP_MARK = '91';
 const SIGNATUREREQUESTTYPE_WITHDRAWSTEP_WITHDRAW = '92';
 const SIGNATUREREQUESTTYPE_CLAIM_GAS = '94';
 
-let lastUTXOWithdrawn;
+const DBG_LOG = false;
```

Comment:
What is this? Would this be better in an environment variable that is set during the build?

Reply:
Yes, we can do that later though.